generator/test_case_generator.go

```diff
@@ -179,10 +179,16 @@ var testTypeToTestConfig = map[string][]testConfig{
 			targets: map[string]map[string]struct{}{"metadataEnabled": {"enabled": {}}},
 		},
 	},
+	"eks_addon": {
+		{
+			testDir: "./test/gpu",
```

Comment:
You'll need to give it a `terraformDir` so it runs the terraform script in the `gpu` directory.